standalone-metastore/metastore-server/pom.xml

```diff
@@ -272,18 +276,22 @@
     <dependency>
       <groupId>org.datanucleus</groupId>
       <artifactId>datanucleus-api-jdo</artifactId>
+      <version>6.0.0-release</version>
```

Comment:
why explicit hardcoded version is required? If we have defined them in properties in pom.xml.